src/main/java/gc/cafe/domain/BaseEntity.java

```diff
@@ -0,0 +1,22 @@
+package gc.cafe.domain;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.EntityListeners;
+import jakarta.persistence.MappedSuperclass;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import java.time.LocalDateTime;
+
+@MappedSuperclass
+@EntityListeners(AuditingEntityListener.class)
+public abstract class BaseEntity {
+
+    @CreatedDate
+    private LocalDateTime createdAt;
```

Comment:
```suggestion
    @Column(updatable = false)
    @CreatedDate
    private LocalDateTime createAt;
```

생성 일자는 업데이트 못하게 막아둘 수 있습니다!